cmd/account/console.go

```diff
@@ -168,8 +166,8 @@ func (o *consoleOptions) run() error {
 	consoleURL, err := aws.RequestSignInToken(
 		awsClient,
 		&o.consoleDuration,
-		awsSdk.String(sessionName),
-		awsSdk.String(targetRoleArn.String()),
+		&sessionName,
+		pointer.String(targetRoleArn.String()),
```

Comment:
It's technically the same thing, but I'm wondering if it makes more sense for this to be `awsSdk.String` where `awsSdk` is now `github.com/aws/aws-sdk-go-v2/aws`

Reply:
the v2 sdk got rid of that package entirely, so I just made these all `pointer.<>` everywhere applicable.

Reply:
Sorry, my first comment misread this.

I'm okay with either, but my preference was to use the standard lib for an operation like this. To compare the functions' source code:

```
AWS SDK V2:
func String(v string) *string {
	return ptr.String(v)
}
// calls down to...
func String(v string) *string {
	return &v
}
```

```
Std Lib:
func String(s string) *string {
	return &s
}
```

So really the same thing is going on under the hood. I'll note that if you (or anyone else 😄 ) feel strongly one way or the other, it'd be extremely easy to do a quick find/replace for `pointer.` through this PR, just let me know!

Reply:
Yeah, if it was in the stdlib, I'd be all for it haha. More that it's been changed to use `"k8s.io/utils/pointer"`?

Reply:
This is a miss by me actually - for some reason I thought this was the `golang.org/x/tools/go/pointer` package 🤦🏻 

Let me revert all of these to use the aws sdk.